### DISCUSSION TITLE
Do math in text field (no input)

### DISCUSSION THREAD

**Discussion by andreapx** [ToolJet/ToolJet#7193] [Q&A] [answered]
Hi all,
I need to multiply 2 valued in a text field like this:
`Result: {{queries.mysql_specs.data[0].field_1}}*{{components.base.value}}`
Is it possible? How can I do that?
Thanks!

**Comment by withshubh** [comment, marked as answer]
Try this: `{{queries.mysql_specs.data[0].field_1 * components.base.value}}`

**Reply by andreapx** [reply]
If I have a division, how can I round the result?
I see that it's possible to use JavaScript in the query, but how to use it in a text field?

**Reply by withshubh** [reply]
Use this `{{Number((queries.mysql_specs.data[0].field_1 / components.base.value).toFixed(2))}}`
This will round off the result to 2 decimal places.

**Reply by andreapx** [reply]
Thanks!
I'm starting to understand how to use JavaScript in ToolJet.

**Comment by andreapx** [comment]
Thanks a lot!!!
